### DISCUSSION TITLE
Black spots on neovim bar

### DISCUSSION THREAD

**Discussion by PaxSov** [arcticicestudio/nord#210] [Help & Support]
On my bar the are places where the colorscheme is not applyed:
screenshot:
![2020-04-20-141331_1920x1080_scrot](https://user-images.githubusercontent.com/63252513/79756019-8787f500-8309-11ea-9d4f-841eee7e98b3.png)
I am using a slightly modified version of this config:
https://github.com/rafi/vim-config
I am on neovim.

also how can I change my neovim config and still easily switch back to the old one?

**Comment by arcticicestudio** [comment]
Hi @e3172 :wave:

In order to ensure the problem is related to the _Nord_ theme you need to reproduce the problem with a minimal and clean configuration. This can be done by creating new configuration file with any name (e.g. `testrc`) and run vim with the following command: `vim -u testrc`

For me as a theme author it's not possible to know if the theme is the root cause of the problem when using such heavyweight configuration bootstrap/framework projects like [rafi/vim-config][] that set tons of configurations and load many plugins where each line of code might break the theme.
I'm also need to know which status bar is used since there are many like e.g. [lightline.vim][] or [vim-airline][].

Here's an minimal test configuration you can use to try to reproduce the problem (supposed you're using [vim-plug][]):

```vim
call plug#begin(expand('~/.vim/plugged'))
Plug 'arcticicestudio/nord-vim'
call plug#end()

set nocompatible
if (has("termguicolors"))
  set termguicolors
endif

syntax enable
colorscheme nord
```

Another guess from me is, based on your screenshot, that your system is missing font types that support emojis and symbols for terminals. Maybe check out projects like [ryanoasis/nerd-fonts][] in order to get compatible fonts.

[vim-plug]: https://github.com/junegunn/vim-plug
[rafi/vim-config]: https://github.com/rafi/vim-config
[ryanoasis/nerd-fonts]: https://github.com/ryanoasis/nerd-fonts
[lightline.vim]: https://github.com/itchyny/lightline.vim
[vim-airline]: https://github.com/vim-airline/vim-airline

**Comment by PaxSov** [comment]
This also happens with your minimal vim configureation, I will try using the nerd fonts

**Comment by jvoisin** [comment]
@PaxSov ping :)

**Comment by svengreb** [comment]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.
